espanol/templo/grito/grito.md

```diff
@@ -1,2 +1,8 @@
 "Fuego en el Templo..."
-Las columnas se vienen abajo.
\ No newline at end of file
+Las columnas se vienen abajo.
+
+El ejercito que se aproximaba, era un ejército enemigo que había comenzado un ataque. 
+
+[El consejero mandó desalojar el templo](/espanol/templo/grito/desalojar/desalojar-templo.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.